kakao-gift/src/stories/Button/Button.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import './button.css'
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * How large should the button be?
+   */
+  size?: 'small' | 'medium' | 'large' | 'responsive'
+  /**
+   * Theme of the button
+   */
+  theme?: 'kakao' | 'outline' | 'black' | 'lightGray'
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({
+  size = 'medium',
+  theme = 'kakao',
+  type = 'button',
+  disabled = false,
```

Comment:
disabled 는 기본 어트리뷰트이기 때문에 생략해도 좋겠네요!

```suggestion
  disabled = false,
```